schema.sql

```diff
@@ -0,0 +1,99 @@
+CREATE TABLE region (
+  id uuid PRIMARY KEY,
```

Comment:
This isn't going to auto generate. According to http://www.starkandwayne.com/blog/uuid-primary-keys-in-postgresql/ and my usage, the following should work:

```
id UUID PRIMARY KEY DEFAULT gen_random_uuid(),
```

Note that the following must be run once on the database to enable that feature:
```
CREATE EXTENSION pgcrypto;
```